### DISCUSSION TITLE
Add image.png to issue in Github using Github for Dart

### DISCUSSION THREAD

**Discussion by AlexandreMarkus** [SpinlockLabs/github.dart#241] [Q&A]
```dart

// file = image.png

await GitHub(auth: Authentication.withToken('xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx'))
             .issues
             .create(RepositorySlug.full("myOrganisation/myRepo"),
            IssueRequest(
                     title: 'New Issue ',
                   body: " ![screenshot]($file)"));
```
I am trying to add the image to my issue but my image doesn't show in Github.

**Comment by robrbecker** [comment]
I assume you've already uploaded the image to github or somewhere public and `$file` holds the URL to access it?

**Reply by AlexandreMarkus** [reply]
Right now my picture is on the local device.
I assume I should upload it first?
Then I should be able to see the image in my issue, is that right?

**Reply by robrbecker** [reply]
https://stackoverflow.com/questions/28703544/create-an-issue-with-image-via-github-api
That article gives some more details. I think your options are add the file to the repo, embed the whole images as a data uri, or upload it to some other place and provide the url in the markdown.

More info on data uri https://css-tricks.com/data-uris/
